common/containers/Tabs/SignAndVerifyMessage/components/SignMessage/index.tsx

```diff
@@ -64,7 +64,7 @@ export class SignMessage extends Component<Props, State> {
               onSignMessage={this.onSignMessage}
             />
           )}
-          <WalletDecrypt hidden={unlocked} />
+          <WalletDecrypt hidden={unlocked} disabledWallets={['trezor']} />
```

Comment:
My preference, if possible, would be to either A) maintain a dictionary of wallet disable arrays keyed by reason (e.g. `DISABLED_WALLETS.CANT_SIGN`) or B) make the props booleans for reasons why to disable them `hideWalletsThatCantSign={true}`. Support for these features is subject to change, and it'd be nice to have one canonical place to track this. I also can see a need for wallets that are read-only, wallets that don't support offline etc.

Reply:
Ah, didn't see that this was merged. If there comes a time I need to adjust this, I hope you don't mind that I implement this! I'll probably need to rework this a bit for the new unlock UI.